packages/at_client/lib/src/client/remote_secondary.dart

```diff
@@ -107,14 +107,8 @@ class RemoteSecondary implements Secondary {
     // ignore: prefer_typing_uninitialized_variables
     var verbResult;
     try {
-      logger.finer(logger.getLogMessageWithClientParticulars(
```

Comment:
why have you removed the logger statements?

Reply:
executeAndParse() in turn calls executeVerb() which also has the logs that I removed. Previously, if executeAndParse() was called, the same log was printed twice. This way(after removal), the log is only printed once.